### PR TITLE
Add pypandoc recipe

### PR DESCRIPTION
**Pull request by janschulz**
I'm upstream maintainer and adding it here would make my work much easier :-)

### CODE REVIEW

File: recipes/pypandoc/build.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install --single-version-externally-managed  --record record.txt
```
Comment (ocefpaf) [inline recipes/pypandoc/build.sh]: @janschulz remove both `bld.bat` and `build.sh` and add the `script: python setup.py install --single-version-externally-managed  --record record.txt` in the build field of the yaml file.

File: recipes/pypandoc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: pypandoc
+  version: 1.1.3
+
+source:
+  git_url: https://github.com/bebraw/pypandoc.git
+  git_tag: v1.1.3
```
Comment (ocefpaf) [inline recipes/pypandoc/meta.yaml]: Can you use the tarball and add a `sha256`/`md5`?

Comment (janschulz) [inline recipes/pypandoc/meta.yaml]: can I use the git hash? Or do release files on github work as well?

File: recipes/pypandoc/build.sh (hunk 1 above)
Comment (janschulz) [inline recipes/pypandoc/build.sh]: done

File: recipes/pypandoc/meta.yaml (hunk 2 above)
Comment (janschulz) [inline recipes/pypandoc/meta.yaml]: ok, github works: done

**Comment by msarahan** [comment]
Thanks for switching to a tarball.  IMHO, a git hash would also be fine.  Tags are not, because people can change what a tag points to.

File: recipes/pypandoc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+about:
+  home: https://github.com/bebraw/pypandoc
+  license: MIT
+  summary: 'Thin wrapper for "pandoc"'
+
+extra:
+  recipe-maintainers:
+    - janschulz
+
```
Comment (jakirkham) [inline recipes/pypandoc/meta.yaml]: nit: strip extra newline. please keep one terminal newline.

File: recipes/pypandoc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    - pandoc
+  run:
+    - python
+    - pandoc
+
+test:
+  imports:
+    - pypandoc
+  commands:
+    - python -c "import pypandoc; print(pypandoc.get_pandoc_version())"
```
Comment (jakirkham) [inline recipes/pypandoc/meta.yaml]: Do you embed your test suite?

File: recipes/pypandoc/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.3" %}
+package:
+  name: pypandoc
+  version: {{ version }}
+
+source:
+  fn: pypandoc-{{ version }}.zip
+  url: https://github.com/bebraw/pypandoc/archive/v{{ version }}.zip
```
Comment (jakirkham) [inline recipes/pypandoc/meta.yaml]: Is it on PyPI?

**Comment by jakirkham** [comment]
Pretty awesome @janschulz! A few nitpicky things, but otherwise looking pretty good.

File: recipes/pypandoc/meta.yaml (hunk 4 above)
Comment (janschulz) [inline recipes/pypandoc/meta.yaml]: It's in the github zip, but I think we don't include it in the pypi upload. It's small enough to be run each time the package is uploaded... Let's see ...

File: recipes/pypandoc/meta.yaml (hunk 3 above)
Comment (janschulz) [inline recipes/pypandoc/meta.yaml]: done

File: recipes/pypandoc/meta.yaml (hunk 5 above)
Comment (janschulz) [inline recipes/pypandoc/meta.yaml]: jein: it is, but I don't want to take it from there as it currently contains an old pandoc download by accident and only the next release will be fixed...

**Comment by janschulz** [comment]
This will be interesting: I'm not sure if we run the testsuite on OSX :-)

**Comment by janschulz** [comment]
> ./.CI/build_all: line 25: conda-build-all: command not found

???

**Comment by ocefpaf** [comment]
LGTM. Thanks @janschulz!

**Pull request merged**